x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authz/interceptor/BulkShardRequestInterceptor.java

```diff
@@ -43,32 +43,33 @@ public BulkShardRequestInterceptor(ThreadPool threadPool, XPackLicenseState lice
     public void intercept(RequestInfo requestInfo, AuthorizationEngine authzEngine, AuthorizationInfo authorizationInfo,
                           ActionListener<Void> listener) {
         boolean shouldIntercept = licenseState.isSecurityEnabled();
-        var licenseChecker = new MemoizedSupplier<>(() -> licenseState.checkFeature(Feature.SECURITY_DLS_FLS));
+        var featureUsageChecker = new MemoizedSupplier<>(() -> licenseState.checkFeature(Feature.SECURITY_DLS_FLS));
```

Comment:
The previous name seems more appropriate to me. The purpose is to check the license allows the feature, not to track usage.

Reply:
I've reverted the name change.